### PR TITLE
Suggest company affiliation

### PR DESCRIPTION
**Pull request by gorshunovr**

### CODE REVIEW

**Comment by imachug** [comment]
At the very least, let's not add this untranslated text to translations. Even then, I still haven't decided if asking for affilition is a good idea. If the company wants to support RMS, they can create a PR with their company name. If it's your personal opinion, then it doesn't really matter whom your work for. (?)

**Comment by shenlebantongying** [comment]
We can add notice like this
> (For individual signatures, institutional affiliation is provided for identification purposes only and does not constitute endorsement.)

**Comment by imachug** [comment]
@nukeop's just pointed out that we already request affiliation here:

> If you are able to, please use your real name and add projects and affiliatied organizations in parentheses.

**Comment by gorshunovr** [comment]
> We can add notice like this
> 
> > (For individual signatures, institutional affiliation is provided for identification purposes only and does not constitute endorsement.)

No, not exactly this one, as it seems to be copied from rms-open-letter, and their license prevents us to reuse their exact wording.

**Comment by shenlebantongying** [comment]
I think this is good suggestion, since people would usually just copy the format. Not to mention that the open letter gatherd damed organizations.

I suggest change `(your company)` to → `(your organization or company)`, since there are people are KDE or archlinux contributers rather than a company.

or `(optimal descriptions)`

**Comment by gorshunovr** [comment]
> At the very least, let's not add this untranslated text to translations.

We can't ask for translations right here and we don't contract translators. Should be OK to be amended later by native speakers via PRs.

**Comment by gorshunovr** [comment]
> or `(optimal descriptions)`

How about `(optional organization or company)`?

**Comment by shenlebantongying** [comment]
How about just change the English one, then post an issue.
If someone noticed, they will help.

No one ask for any translations, but still we have 18 already.

File: README_KO.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
 서명하시려면, **[이곳](https://github.com/rms-support-letter/rms-support-letter.github.io/new/master/_data/signed)을 클릭하시고** `<username>.yaml` (`<username>`을 귀하의 이름으로 변경하십시오) 이라는 파일을 만드시로 다음과 같은 내용을 작성해주십시오.
 
 ```yaml
-name: <your name here>
+name: <your name here (optional organization or company)>
```
Comment (gorshunovr) [inline README_KO.md:6]: Left suggestion in English only because it is already used in the example.

**Review by gorshunovr — COMMENTED** [review]
Updated.

File: README_RU.md [hunk 2]
```diff
@@ -4,15 +4,15 @@
 Для того, чтобы подписать письмо, создайте в директории `_data/signed/` файл `<username>.yaml` со следующим содержимым:
 
 ```yaml
-name: <укажите Ваше имя здесь>
+name: <укажите Ваше имя здесь (опционально, укажите проекты/организации)>
```
Comment (imachug) [inline README_RU.md:7]: ...'в которых вы участвуете'?

Comment (gorshunovr) [reply README_RU.md:7]: Длинно. Замучаешься стирать.

**Comment by imachug** [comment]
Thanks everyone!

**Pull request merged**